src/Bicep.Cli/Arguments/PublishArguments.cs

```diff
@@ -33,6 +34,8 @@ public PublishArguments(string[] args) : base(Constants.Command.Publish)
                         break;
 
                     case "--documentationuri":
+                        io.WriteParameterDeprecationWarning("--documentationUri", "--documentation-uri");
```

Comment:
I believe this parameter is used in AzCLI & AzPwsh. To avoid printing a warning that the user can't fix, should we have an interim release without this message, and then introduce a version check in AzCLI/Pwsh?

Reply:
Good point. Will comment out this for now.

Reply:
Done.